docs/source/install.rst

```diff
@@ -19,6 +19,8 @@ The following third party libraries (TPLs) are used by DTK:
 +------------------------+------------+---------+
 | Trilinos               | Required   | 12.X    |
 +------------------------+------------+---------+
+| Benchmark              | Required (if examples are enabled)  | 1.4     |
```

Comment:
Change to Google Benchmark?

Reply:
Will do.  I was not sure.  They always refer to it as "Benchmark" but if we say Google Benchmark it might be easier to find.